### PR TITLE
feat: Branding

### LINKED ISSUE #674: [Docs]: Branding
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Engineering Ticket Link

https://github.com/appsmithorg/appsmith/issues/15828

### Release Date

15 December 2022 / v1.8.13

### First Draft

#680 

### PRD

https://www.notion.so/appsmith/Branding-and-whitelabeling-798a3d121bee4922b8b20d00e5c07572

### Test plan/cases

_No response_

### Use cases or user requests

_No response_

### PR DESCRIPTION
**Pull request by jsartisan**
Adds docs for branding feature

Fixes #674

The docs will be as follows:

- The placement is correct - Keep it under Advanced concepts
- Add new images and annotations as per docs standard
- Add a short video and steps to explain how to upload logo, favicon (based on loom video shared by @dilippitchika) and how it's displayed on Appsmith
- Take a sample logo which could just be a drawing or a sample image placeholder

### CODE REVIEW

**Comment by jnikhila** [comment]
@jsartisan - Could you please sign and accept the CLA?

**Comment by jnikhila** [comment]
> Adds docs for branding feature
> 
> Fixes #674

The docs will be as follows:
* The placement is correct - Keep it under Advanced concepts
* Add new images and annotations as per docs standard
* Add a short video and steps to explain how to upload logo, favicon (based on loom video shared by @dilippitchika) and how it's displayed on Appsmith
* Take a sample logo which could just be a drawing or a sample image placeholder

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/docs/advanced-concepts/branding.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+To configure Appsmith to show your brand, follow these steps:
+
+<VideoEmbed host="youtube" videoId="aK-JHRFNUrU" title="Branding | Demo" caption="Set up Appsmith to show your own branding" /> 
+
+1. From your Appsmith dashboard, navigate to the [Admin Settings](/getting-started/setup/instance-configuration/admin-settings). You must to have Administrator privileges to see access this area.
+2. Click on the **Branding** tab to open branding settings.
+3. To upload your logo, click in the **Logo** box and browse to select the image file you'd like to upload. Appsmith supports .svg, .png, and .jpg files up to 2 MB in size.
+4. To upload a favicon, click in the **Favicon** box and browse to select the file you'd like to upload. You may provide a file of type .ico, .png, or .jpg which shouldn't be larger than 32x32 pixels.
+5. In the **Color** section, each of the three bars of color represent the primary, background, and font colors. Click the one you'd like to customize, and enter a hex color code in the input box. Alternatively, you can click the color preview circle in the box to open a color-picker window and select it that way.
+<!--
```
[the remark this answers is not in the document]
Comment (spark-c) [reply website/docs/advanced-concepts/branding.md:34]: Uncommented the tip, was previously commented due to technical difficulty

File: website/docs/advanced-concepts/branding.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+# Branding
+
+Custom branding for Appsmith is a feature that allows app builders to customize the appearance of their Appsmith workspace and apps with their own logo and color palette. The ensures that the end-users of the app have a more familiar and cohesive experience, with the login screen, invite emails, and error pages matching the branding of the user's company. 
+
+:::info
+Branding is only available on self-hosted, [**Business Edition**](https://www.appsmith.com/pricing) instances of Appsmith.
+:::
+
```
[the remark this answers is not in the document]
Comment (danciaclara) [reply website/docs/advanced-concepts/branding.md:13]: Is this subheading even required?  IMO it looks a bit odd with just one on the right. if there are no other sections on this page, let's avoid this as well. 
Screenshot 2022-12-16 at 17 20 35

**Review by danciaclara — APPROVED** [review]

File: website/docs/advanced-concepts/branding.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jnikhila) [reply website/docs/advanced-concepts/branding.md:13]: That's fine if there is only one heading. The page runs with lots of text without a heading, and it feels like too much of information. And we are writing about configuring branding here so the heading looks apt.

**Review by jnikhila — APPROVED** [review]

**Comment by jnikhila** [comment]
@dilippitchika - The PR is ready to be merged. Can we publish it?

**Pull request merged**